net-analyzer/gr-fosphor/gr-fosphor-9999.ebuild

```diff
@@ -22,41 +22,35 @@ fi
 
 LICENSE="GPL-3+"
 SLOT="0"
-IUSE="glfw qt5"
+IUSE="glfw"
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
-RDEPEND="
+RDEPEND="${PYTHON_DEPS}
 	dev-libs/boost:=
 	dev-libs/log4cpp
-	media-libs/freetype
-	>=net-wireless/gnuradio-3.9:0=[qt5,${PYTHON_SINGLE_USEDEP}]
 	$(python_gen_cond_dep 'dev-python/pygccxml[${PYTHON_USEDEP}]')
+	media-libs/freetype
+	>=net-wireless/gnuradio-3.9:0=[${PYTHON_SINGLE_USEDEP}]
 	glfw? ( >=media-libs/glfw-3 )
 	virtual/opencl
 	virtual/opengl
-	${PYTHON_DEPS}
-	qt5? (
-		dev-qt/qtcore:5
-		dev-qt/qtgui:5
-		dev-qt/qtopengl:5
-		dev-qt/qtwidgets:5
-	)"
+"
 DEPEND="${RDEPEND}"
 BDEPEND="$(python_gen_cond_dep 'dev-python/pybind11[${PYTHON_USEDEP}]')"
 
-PATCHES=( "${FILESDIR}"/${PN}-0.0_p20200131-htmldir.patch
-		  "${FILESDIR}"/${PN}-0.0_p20210108-fix-use.patch )
+PATCHES=(
+	"${FILESDIR}"/${PN}-0.0_p20200131-htmldir.patch
+	"${FILESDIR}"/${PN}-0.0_p20210108-fix-use.patch
+)
 
 src_prepare() {
 	cmake_src_prepare
 
 	# adapt python bindings to use flags
 	use glfw || sed -i -e "s#bind_glfw_sink_c(m)##" \
-					"${S}"/python/bindings/python_bindings.cc ||die
+		python/bindings/python_bindings.cc || die
 	use qt5 || sed -i -e "s#bind_qt_sink_c(m)##" \
```

Comment:
Does it make sense to have a 'use qt5' here when we drop 'qt5' from IUSE?

Reply:
@dl1jbe thanks for your review, much appreciated